### DISCUSSION TITLE
Disabled dates and time

### DISCUSSION THREAD

**Discussion by PrasadChinwal** [GuavaCZ/calendar#31] [Q&A] [answered]
Hi, 

I wanted to check if it is possible to disable some dates and time in the calendar view. 

Use Case: I have a `timeGridWeek` calendar and have `availability schedules` of user. I want to create/edit events only on the availability dates and time. 

Ex. On monday User 1 is available from 9 am to 5 pm. So we disable the remaining hours in the `timeGridWeek`.

**Comment by lukas-frey** [comment, marked as answer]
Check vkurko/calendar options. You can set slotMinTime and slotMaxTime to define the available dates via the calendar.
